package.json

```diff
@@ -29,7 +29,7 @@
     "build:core": "bili",
     "prebuild:plugins": "rimraf lib/plugins/*.js lib/plugins/*.js.map",
     "build:plugins": "npm-run-all -s build:plugin:*",
-    "build:plugin:authenticate": "PLUGIN=authenticate bili",
+    "build:plugin:auth": "cross-env PLUGIN=auth bili",
```

Comment:
`auth` plugin gets built into the final bundle anyway. Why does it need to be built separately?

Reply:
I removed it for now as a first solution, and also asked for an [advice on how to do it better](https://github.com/MunifTanjim/node-bitbucket/pull/78#issuecomment-787420622).

Reply:
I removed it for now.